packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -171,12 +169,13 @@ void main() {
                     child: Text(value),
                   );
                 }).toList(),
-                onChanged: (String newValue) {
-                  setState(() {
-                    value = newValue;
-                  });
+                onChanged: (String? newValue) {
+                  if (newValue != null)
+                    setState(() {
+                      value = newValue;
+                    });
```

Comment:
Should we just make `value` nullable? It seems that a null value is ok for the DropdownButtonFormField?